src/dialogs/ShuffleDialog.java

```diff
@@ -313,6 +371,30 @@ private void jButton2ActionPerformed() {//GEN-FIRST:event_jButton1ActionPerforme
             "Advanced Shuffle Editor", JOptionPane.INFORMATION_MESSAGE);
     }//GEN-LAST:event_jButton1ActionPerformed
 
+    private void jButton3ActionPerformed() {//GEN-FIRST:event_jButton1ActionPerformed
```

Comment:
these comments don't match the method names

Reply:
This will be fixed later, on all GUIs (just like adding more descriptive names)